### DISCUSSION TITLE
How to create a minified file (or single js)like file from CDN, from the library in node_modules

### DISCUSSION THREAD

**Discussion by durga73** [webpack/webpack#18270] [Q&A]
We are creating a Micro FrontEnd apps with module federation. We have a custom library that is used in all the apps this library is getting added to each app bundle which is causing the increase in bundle size. We are planning to externalize the library but when we do import from index.html it asks for all the dependencies of that app in import maps. How to create a single js file with all the dependencies of that library and use it as external and use it in import maps.

Thanks in advance
Durga

**Comment by alexander-akait** [comment]
Just use multi compiler mode, bundle it without asyncChunks (look at output options) and use the `external` option to import it in your app

**Reply by durga73** [reply]
can you share any example.

**Reply by alexander-akait** [reply]
There is an option https://webpack.js.org/configuration/output/#outputasyncchunks
